tcc/src/main/java/io/seata/rm/tcc/interceptor/ActionContextUtil.java

```diff
@@ -96,9 +98,7 @@ public static void getAllField(Class<?> interFace, List<Field> fields) {
             return;
         }
         Field[] field = interFace.getDeclaredFields();
-        if (field != null) {
-            fields.addAll(Arrays.asList(field));
-        }
+        fields.addAll(Arrays.asList(field));
```

Comment:
Why remove field != null?

Reply:
Because interFace.getDeclaredFields() can never empty, if there is no field in the interFace default returned for an empty array, instead of null.